### PR TITLE
SCRUM-145: PR validation setup (through Github actions)

### PR DESCRIPTION
**Pull request by mluypaert**
PR created to test if a Github actions workflow gets triggered (config files to do so are in branch to be merged).

### CODE REVIEW

**Review by oblodgett — APPROVED** [review]
@mluypaert is this just a test? or do you want to push the code in?

File: .github/workflows/github-actions-PR-demo.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        aws-region: us-east-1
+    - name: Amazon ECR login
+      id: login-ecr
+      uses: aws-actions/amazon-ecr-login@v1
+    - name: Check out repository code
+      uses: actions/checkout@v2
+    - name: Build, tag, and push image to Amazon ECR
+      env:
+        ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+        ECR_REPOSITORY: agr_curation_prototype
```
Comment (oblodgett) [inline .github/workflows/github-actions-PR-demo.yml:42]: We have changed the repo name back to `agr_curation` so we might want to change the ECR repo name.

**Review by oblodgett — APPROVED** [review]

**Comment by mluypaert** [comment]
This is mainly a PR for testing purpose, I'm building a similar branch for GoCD pipelines-as-code testing and then once testing is completed for both we can decide which one to procede with and merge. The PR validation steps to be taken (and the `Dockerfile` I created for it) are to be reviewed and agreed upon before merging, so the PRs after merging get the correct validation.

**Comment by mluypaert** [comment]
@oblodgett I re-requested your review, as the code has evolved and this PR is now intended to get merged in order to enable automatic code validation on all PRs.
I included a statement to ignore `.gitignore` and `README.md` so no PR validation runs will get triggered when no other files got updated.

I also found a way to get a list of the files which have been updated in a PR, which could later be used for targeted code validation as required. See the `verify-test-api/Report files updated in PR` output.

I will merge this PR and update the repository settings to require successfull execution of the `PR validation` github actions workflow (created through this PR) once this PR is approved again.

**Review by oblodgett — APPROVED** [review]

**Pull request merged**